openapi/api.github.com/index.json

```diff
@@ -1431,42 +1402,56 @@
       "get": { "$ref": "operations/users/get-authenticated.json" },
       "patch": { "$ref": "operations/users/update-authenticated.json" }
     },
-    "/user/blocks": { "get": { "$ref": "operations/users/list-blocked.json" } },
+    "/user/blocks": {
+      "get": { "$ref": "operations/users/list-blocked-by-authenticated.json" }
+    },
     "/user/blocks/{username}": {
       "get": { "$ref": "operations/users/check-blocked.json" },
       "put": { "$ref": "operations/users/block.json" },
       "delete": { "$ref": "operations/users/unblock.json" }
     },
     "/user/email/visibility": {
       "patch": {
-        "$ref": "operations/users/toggle-primary-email-visibility.json"
+        "$ref": "operations/users/set-primary-email-visibility-for-authenticated.json"
       }
     },
     "/user/emails": {
-      "get": { "$ref": "operations/users/list-emails.json" },
-      "post": { "$ref": "operations/users/add-emails.json" },
-      "delete": { "$ref": "operations/users/delete-emails.json" }
+      "get": { "$ref": "operations/users/list-emails-for-authenticated.json" },
+      "post": { "$ref": "operations/users/add-email-for-authenticated.json" },
+      "delete": {
+        "$ref": "operations/users/delete-email-for-authenticated.json"
+      }
     },
     "/user/followers": {
       "get": {
         "$ref": "operations/users/list-followers-for-authenticated-user.json"
       }
     },
     "/user/following": {
-      "get": { "$ref": "operations/users/list-followed-by-authenticated.json" }
+      "get": {
+        "$ref": "operations/users/list-people-authenticated-follows.json"
+      }
```

Comment:
needs deprecation, or maybe operatrion ID override, I think the previous one was more clear